.github/workflows/main.yml

```
@@ -0,0 +1,51 @@
+name: build
+on:
+ push:
+ pull_request:
+ schedule:
+#Every 50 days at midnight 
+    - cron:  "0 0 1/600 * *"
+
+jobs:
+  compilejobFedora:
+    name: APFEL_on_Fedora
+    runs-on: ubuntu-latest
+    container:
+        image: fedora:latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 3
+    - name: Install 
+      run: |
+           uname -a 
+           cat /etc/issue
+           yum -y install  gcc gcc-c++ gcc-gfortran make wget which cmake cmake-data cmake-filesystem lhapdf lhapdf-devel swig python-devel
+    - name: Compile
+      run: |
+          cmake -S . -B BUILD -DCMAKE_INSTALL_PREFIX=$(pwd)/../INSTALL
+          cmake --build BUILD -j 2
```

If `-j` has the same meaning of autotools, in principle you can avoid specifying a number at all, and it will autoselect the number of available cores
```suggestion
          cmake --build BUILD -j
```